src/main/java/org/jenkinsci/plugins/saml/SamlFormValidation.java

```diff
@@ -0,0 +1,81 @@
+package org.jenkinsci.plugins.saml;
+
+import hudson.util.FormValidation;
+import org.apache.commons.lang.StringUtils;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import static org.jenkinsci.plugins.saml.SamlSecurityRealm.*;
+
+public class SamlFormValidation extends FormValidation {
```

Comment:
Might want to add `@Restricted(NoExternalUse.class)` to make this class local to the module. Or you could make it package-private since it doesn't seem to be used outside this package anyways.

Reply:
Thank you for your feedback